core-tests/shared/src/test/scala/zio/internal/ExecutorSpec.scala

```diff
@@ -35,44 +37,51 @@ object TestExecutor {
   }
 }
 
-class ExecutorSpec extends Specification {
-  def is =
-    "ExecutorSpec".title ^ s2"""
-      Create the default unyielding executor and check that:
-        When converted to an EC, it reports Throwables to stdout                   $exec1
-
-      Create an executor that cannot have tasks submitted to and check that:
-        It throws an exception upon submission                                     $fail1
-        When converted to an ExecutionContext, it throws an exception              $fail2
-        When created from an EC, throw when fed an effect                             $fail3
-
-      Create a yielding executor and check that:
-        Runnables can be submitted                                                 $yield1
-        When converted to an ExecutionContext, it accepts Runnables                $yield2
-        When created from an EC, must not throw when fed an effect                    $yield3
-
-      Create an unyielding executor and check that:
-        Runnables can be submitted                                                 $unyield1
-        When converted to an ExecutionContext, it accepts Runnables                $unyield2
-    """
-
-  def exec1 = {
-    val t = new CheckPrintThrowable
-    TestExecutor.failing.asEC.reportFailure(t)
-    t.printed must beTrue
-  }
-
-  def fail1 = TestExecutor.failing.submitOrThrow(() => ()) must throwA[RejectedExecutionException]
-  def fail2 = TestExecutor.failing.asEC.execute(() => ()) must throwA[RejectedExecutionException]
-  def fail3 =
-    Executor
-      .fromExecutionContext(1)(TestExecutor.badEC)
-      .submitOrThrow(() => ()) must throwA[RejectedExecutionException]
-
-  def yield1 = TestExecutor.y.submitOrThrow(() => ()) must not(throwA[RejectedExecutionException])
-  def yield2 = TestExecutor.y.asEC.execute(() => ()) must not(throwA[RejectedExecutionException])
-  def yield3 = Executor.fromExecutionContext(1)(TestExecutor.ec).submit(() => ()) must beTrue
-
-  def unyield1 = TestExecutor.u.submitOrThrow(() => ()) must not(throwA[RejectedExecutionException])
-  def unyield2 = TestExecutor.u.asEC.execute(() => ()) must not(throwA[RejectedExecutionException])
-}
+object ExecutorSpec
+    extends ZIOBaseSpec(
+      suite("Create the default unyielding executor and check that:")(
```

Comment:
Can you make the root suite just called`ExecutorSpec` and contains the other suites?

Reply:
ok, I'll do it